src/bentoml/_internal/cloud/__init__.py

```diff
@@ -0,0 +1,119 @@
+from abc import ABC, abstractmethod
```

Comment:
```suggestion
from __future__ import annotations

from abc import ABC, abstractmethod
```

add annotations support